GUI/dashboard.py

```diff
@@ -132,7 +132,7 @@ def myClick():
     s="abcd@gmail.com"  # for testing purposes
     mylabel = Label(second_frame, text=str(counter)+' '+'You Entered: '\
                                +input_box.get()+'\nRequestor: '+' '+str(s)+'\n')
-    mylabel.grid(column=3, row=6)
+    mylabel.grid(column=3, row=5+counter)
```

Comment:
The latest entered name/email should be at top, it's still printing the last entered entry in the last. The number before each text i.e. counter sort of gives an idea which order the entered text got printed.